content/contribute/terms-and-conditions.md

```diff
@@ -7,3 +7,43 @@ linked:
 ---
 
 ## HCA Data Contributor Terms and Conditions
+
+### Preliminary terms and conditions for data use and sharing
+
+Version: 
+
+As soon as the official Human Cell Atlas terms and conditions for data contribution are finalised and reviewed by our legal authorities, you will be asked to agree to them. In that instance, we will delete all data you have already submitted to the Human Cell Atlas Data Coordination Platform.
+
+
+1. All data submitted to the Human Cell Atlas Data Coordination Platform will be made
+available for unrestricted public view, redistribution, and reuse. *
```

Comment:
Starting with point 1, there are returns for each row of text. This should be changed to allow a single term to be on 1 line so that wrapping can be handled consistently.